calculator.js

```diff
@@ -1,7 +1,51 @@
 "use strict";
 
-export class Calculator {
+export class Calculator{
   //write your code here
+  constructor(value){
+    this._angka = value || 0;
+  }
+  //
+  set angka(value){
+    this._angka = value
```

Comment:
butuh di revisi sedikit lagi, agar method chaining nya jalan.
yg di return adalah object nya yaitu this, sehingga bisa dipanggil dengan cara : 
console.log("penjumlahan : " + calculator.add(10).sub(2));